paper_search_mcp/server.py

```
@@ -345,6 +345,85 @@ async def read_semantic_paper(paper_id: str, save_path: str = "./downloads") ->
+
+    Args:
+        paper_id: Semantic Scholar paper ID (e.g., "649def34f8be52c8b66281af98ae884c09aef38b")
+        max_results: Maximum number of citing papers to return (default: 20)
+
+    Returns:
+        List of papers that cite the given paper.
+
+    Example:
+        await get_semantic_citations("5bbfdf2e62f0508c65ba6de9c72fe2066fd98138", 10)
+    """
+    async with httpx.AsyncClient() as client:
```

Ruff (0.14.13)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)